src/main/java/org/folio/consortia/service/impl/CustomFieldServiceImpl.java

```diff
@@ -0,0 +1,54 @@
+package org.folio.consortia.service.impl;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+import org.folio.consortia.client.CustomFieldsClient;
+import org.folio.consortia.client.OkapiClient;
+import org.folio.consortia.domain.dto.CustomField;
+import org.folio.consortia.service.CustomFieldService;
+import org.folio.spring.FolioExecutionContext;
+import org.folio.spring.exception.NotFoundException;
+import org.springframework.cache.annotation.Cacheable;
```

Comment:
import order seems to be different from our usual

Reply:
fixed manually for now in all.